js/speakers.js

```diff
@@ -95,4 +96,5 @@ speakersection.innerHTML = `<div class="speakers">
 </div>
 </li>
 </ul>
-</div>`;
\ No newline at end of file
+</div>`;
```

Comment:
## [OPTIONAL]

#### Your code
```js
window.onload = () => {
  speakersection.innerHTML = `<div class="speakers">
<h4>Featured Speakers</h4>
<hr class="orangehr">
<ul class="speakersul">
    <li class="speakercard">
        <img src="${speakerinfo[0].picture}">
        <div class="descrip">
            <h5>${speakerinfo[0].name}</h5>
            <p class="subtitle">${speakerinfo[0].subtitle}</p>
            <hr class="grayhr">
            <p class="speakerdescrip">${speakerinfo[0].description}</p>
        </div>
    </li> <li class="speakercard">
    <img src="${speakerinfo[1].picture}">
    <div class="descrip">
        <h5>${speakerinfo[1].name}</h5>
        <p class="subtitle">${speakerinfo[1].subtitle}</p>
        <hr class="grayhr">
        <p class="speakerdescrip">${speakerinfo[1].description}</p>
    </div>
</li>
    <div class="morebutton ">MORE<i class="material-icons">arrow_drop_down</i></div>
</ul>
<ul class="speakersul mobilecollapse">
<li class="speakercard">
<img src="${speakerinfo[2].picture}">
<div class="descrip">
    <h5>${speakerinfo[2].name}</h5>
    <p class="subtitle">${speakerinfo[2].subtitle}</p>
    <hr class="grayhr">
    <p class="speakerdescrip">${speakerinfo[2].description}</p>
</div>
</li> <li class="speakercard">
<img src="${speakerinfo[3].picture}">
<div class="descrip">
    <h5>${speakerinfo[3].name}</h5>
    <p class="subtitle">${speakerinfo[3].subtitle}</p>
    <hr class="grayhr">
    <p class="speakerdescrip">${speakerinfo[3].description}</p>
</div>
</li> <li class="speakercard">
<img src="${speakerinfo[4].picture}">
<div class="descrip">
    <h5>${speakerinfo[4].name}</h5>
    <p class="subtitle">${speakerinfo[4].subtitle}</p>
    <hr class="grayhr">
    <p class="speakerdescrip">${speakerinfo[4].description}</p>
</div>
</li> <li class="speakercard">
<img src="${speakerinfo[5].picture}">
<div class="descrip">
    <h5>${speakerinfo[5].name}</h5>
    <p class="subtitle">${speakerinfo[5].subtitle}</p>
    <hr class="grayhr">
    <p class="speakerdescrip">${speakerinfo[5].description}</p>
</div>
</li>
</ul>
</div>`; 
```
- [ ]  Good job 👏  👏  on adding speakers when the page loads however, I can see some DRY(Don't Repeat Yourself) code that I can suggest you improve. Imagine if your speakers were 100... would you access them using indexes ?? this could be a long code.
- Instead of getting speakers' information by index, you can just use the `forEach` method to add speakers dynamically.
```js
const speakersul = document.getElementByClassName('speakersul')[0];
speakerinfo.forEach( speacker  => {
     speakerul.innerHTML = +`<li class="speakercard">
        <img src="${speaker.picture}">
        <div class="descrip">
            <h5>${speaker.name}</h5>
            <p class="subtitle">${speaker.subtitle}</p>
            <hr class="grayhr">
            <p class="speakerdescrip">${speaker.description}</p>
        </div>
    </li> `
})
```